signalr-go-server/pkg/signalr/hubprotocol.go

```diff
@@ -18,11 +23,17 @@ type hubMessage struct {
 }
 
 type invocationMessage struct {
-	Type         int           `json:"type"`
-	Target       string        `json:"target"`
-	InvocationID string        `json:"invocationId,omitempty"`
-	Arguments    []interface{} `json:"arguments"`
-	StreamIds    []string      `json:"streamIds,omitempty"`
+	Type         int
+	Target       string
+	InvocationID string
+	Arguments    []interface{}
+	StreamIds    []string
+}
+
+type sendOnlyHubInvocationMessage struct {
```

Comment:
Undo this

Reply:
01e83d66d85ae95bd8278ae2fdd940464c3253cd